### ISSUE TITLE
Add a way to set the default trainer battle transition

### ISSUE THREAD

**Issue by AerunDev** [PokemonWorkshop/PokemonStudio#226]

**Comment by Green00101** [comment]
Hi, I’m interested in working on this issue.

I don’t know much about Ruby yet, so I may need some time to understand the PSDK part.

Would it be okay if I give it a try?

**Comment by AerunDev** [comment]
> Would it be okay if I give it a try?

Of course! Feel free to. If you have some questions about the UI, UX or even the Ruby part, you can [ask on our Discord](https://discord.gg/0noB0gBDd91B8pMk), or even on this issue.

Asking is the better way to reduce the amount of back and forth. 😄

**Comment by atlasdev9999-ctrl** [comment]
Hi Sorry this is squishy from discord strangly it wont let me self assign  but I have started working on it

**Comment by Green00101** [comment]
Thanks for letting me know!
Feel free to keep going with this one. I’ll look for another issue to work on.
